src/content/en/updates/2011/11/Quota-Management-API-Fast-Facts.md

```diff
@@ -8,46 +8,45 @@ book_path: /web/updates/_book.yaml
 # Quota Management API : Fast Facts {: .page-title }
 
 {% include "web/_shared/contributors/agektmr.html" %}
+November 2011
```

Comment:
I put the date specifically because the article is really old. IMO, we *should* have dates on these kinds of articles, because the only indicator of an article's age is in the URL, which is poor accessibility (esp. on mobile).